plugin/trino-pinot/src/main/java/io/trino/plugin/pinot/PinotConfig.java

```diff
@@ -310,4 +310,23 @@ public boolean proxyRestAndGrpcAreRequired()
         }
         return true;
     }
+
+    public String getQueryOptions()
+    {
+        return queryOptions.orElse(null);
+    }
+
+    @Config("pinot.query-options")
```

Comment:
Add info about this new configuration in `pinot.md`.

Reply:
@findinpath Done.